Neos.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst

```diff
@@ -297,11 +297,17 @@ The following options are allowed:
 
     ``inline``
       This section controls the text formatting options the user has available for this property.
+
+      **Note**: When using `inline.editorOptions` anything defined under the legacy `aloha` key for a
```

Comment:
I think we need to agree on whether we want to fix this behavior or leave as is. If we would fix it, then we should say so that this is a temporary bug and will start working soon.

Reply:
It would be good to merge those, with `inline.editorOptions` "winning" over `aloha` in case of a conflict. Then this note could be dropped again. For now, it's important to know, IMHO.